owners/test/index.test.js

```diff
@@ -526,55 +526,4 @@ describe('owners bot', () => {
       await probot.receive({name: 'pull_request_review', payload: review35});
     });
   });
-
-  describe('when there is a failure to run the check', () => {
```

Comment:
This was a relic that should have been removed in the PR that moved error checking into OwnersCheck.